### ISSUE TITLE
Impossible to use with Ionic 3

### ISSUE THREAD

**Issue by plmercereau** [udos86/ng2-dynamic-forms#453]
Hello @udos86,
Congrats on the outstanding work you've done! However I can't find a way to make it work with the lastest version of Ionic:
- Ionic: 3.5.3
- Angular: 4.1.3
- Node: 7.10.0

It shows me an error when trying to load the page with the form: "No provider for DynamicFormService!".
I quickly created [a GitHub rep that reproduces the error](https://github.com/plmercereau/test-ionic3-dynamicform-froms-cratch).

I simply created it with the Ionic CLI:
`ionic start testRepo`
`npm install @ng2-dynamic-forms/core @ng2-dynamic-forms/ui-ionic angular2-text-mask --save`

Then I followed your recommendations (Basic Usage).

My guess is that it is related to the Ionic lazy loading system, but I can't find a way to fix it...
Many, many thanks in advance!

**Comment by udos86** [comment]
@plmercereau Hi, thanks for the nice words!

I made a quick test and everything seems to be working fine with latest Ionic version!

After checking your repo it's obvious why it's not working. You forgot to import the lib into your app module:

```
@NgModule({
  declarations: [
    MyApp,
    HomePage
  ],
  imports: [
    BrowserModule,
    IonicModule.forRoot(MyApp),
    DynamicFormsCoreModule.forRoot(),
    DynamicFormsIonicUIModule
  ],
  bootstrap: [IonicApp],
  entryComponents: [
    MyApp,
    HomePage
  ],
  providers: [
    StatusBar,
    SplashScreen,
    { provide: ErrorHandler, useClass: IonicErrorHandler }
  ]
}) 
```

**Comment by plmercereau** [comment]
Woops
I don't understand how I missed it, in the example I just wrote, and on the project I am working on.
The lazy loading of Ionic 3 allows not to declare the pages in the main NgModule in app.module.ts, but then it is needed to add to the imports DynamicFormsCoreModule.forRoot() and DynamicFormsIonicUIModule to each Ionic page declaration, i.e. I did not find a way to declare your lib globally. I will push the correction into [this repo](https://github.com/plmercereau/pwah-client) if you want to have a look.
Anyway, vielen dank ;)
